packages/types/src/phase0/types/wire.ts

```diff
@@ -33,6 +33,7 @@ export type Ping = Uint64;
 export interface Metadata {
```

Comment:
You'll need to make a `Metadata` type for altair, this one is for phase0 and can't change.

Also, the `MetadataController` will need an additional method / tweaked methods eg: `getPhase0(): phase0.Metadata` and `getAltair(): altair.Metadata` so that we can support both protocols `/eth2/beacon_chain/req/metadata/1/` and `/eth2/beacon_chain/req/metadata/2/`